src/bo4e/bo/lokationszuordnung.py

```diff
@@ -0,0 +1,47 @@
+"""
+Contains Lokationszuordnung class
+"""
+
+from typing import Annotated, Optional
+
+from pydantic import Field
+
+from ..com.zeitspanne import Zeitspanne
+from ..enum.arithmetische_operation import ArithmetischeOperation
+from ..enum.typ import Typ
+from ..utils import postprocess_docstring
+from .geschaeftsobjekt import Geschaeftsobjekt
+
+
+@postprocess_docstring
+class Lokationszuordnung(Geschaeftsobjekt):
+    """
+    Modell für die Abbildung der Referenz auf die Lokationsbündelstruktur. Diese gibt an welche Marktlokationen,
+    Messlokationen, Netzlokationen, technische/steuerbaren Ressourcen an einer Lokation vorhanden sind;
+
+    .. raw:: html
+
+        <object data="../_static/images/bo4e/bo/Lokationszuordnung.svg" type="image/svg+xml"></object>
+
+    .. HINT::
+        `Lokationszuordnung JSON Schema <https://json-schema.app/view/%23?url=https://raw.githubusercontent.com/BO4E/BO4E-Schemas/{__gh_version__}/src/bo4e_schemas/bo/Lokationszuordnung.json>`_
+    """
+
+    typ: Annotated[Optional[Typ], Field(alias="_typ")] = Typ.LOKATIONSZUORDNUNG
+
+    #: Liste mit IDs der referenzierten Marktlokationen
+    marktlokationen: Optional[list[str]] = None
+    #: Liste mit IDs der referenzierten Messlokationen
+    messlokationen: Optional[list[str]] = None
+    #: Liste mit IDs der referenzierten Netzlokationen
+    netzlokationen: Optional[list[str]] = None
+    #: Liste mit IDs der referenzierten technischen Ressourcen
+    technische_ressourcen: Optional[list[str]] = None
+    #: Liste mit IDs der referenzierten steuerbaren Ressourcen
+    steuerbare_ressourcen: Optional[list[str]] = None
+    #: Zeitspanne der Gültigkeit
+    gueltigkeit: Optional[Zeitspanne] = None
+    #: Angabe einer arithmetischen Operation
+    arithmetik: Optional[ArithmetischeOperation] = None
+    #: Code, der angibt wie die Lokationsbündelstruktur zusammengesetzt ist
+    zuordnungstyp: Optional[str] = None
```

Comment:
Hier haben wir länger darüber diskutiert. Wir würden den `zuordnungstyp` hier drin lassen, um die jeweilige Verknüpfung/Richtung abzubilden. Müssen wir nochmal schauen, ob hier Enum oder string sinnvoller ist. 

Zusätzlich soll es einen `lokationsbuendelcode` geben, wie er in der [Edi@Energy Codeliste](https://www.edi-energy.de/index.php?id=38&tx_bdew_bdew%5Buid%5D=2190&tx_bdew_bdew%5Baction%5D=download&tx_bdew_bdew%5Bcontroller%5D=Dokument&cHash=5f6fc1483454cfa305629fe40bd24e16) abgebildet ist. 

Dadurch soll es möglich sein, sowohl über den Code und die Lokationen das Lokationsbündel anzugeben als auch die Struktur direkt abzubilden.

Reply:
Du kannst es erstmal als String implementieren. Ich hab dafür schon ein Ticket formuliert, das bitte in einen Kommentar dazu schreiben.